cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/PackCommandTest.java

```diff
@@ -270,4 +270,32 @@ public void testCustomTargetDirWithRelativePath() throws IOException {
         Assert.assertTrue(customTargetDir.resolve("bala").resolve("foo-winery-any-0.1.0.bala").toFile().exists());
     }
 
+    @Test(description = "Build an empty library module")
+    public void testPackEmptyLibModule() throws IOException {
```

Comment:
```suggestion
    public void testPackEmptyPackage() throws IOException {
```
We do not need to specify that it is a lib. We assume that pack command will be run on lib packages only